code/Backend/flask-backend/app.py

```diff
@@ -294,12 +270,6 @@ def login():
     except Exception as e:
         return jsonify({"error": str(e)}), 500
 
-@app.route('/api/dashboard', methods=['GET', 'POST'])
```

Comment:
Do we need to create a new dashboard page?

Reply:
It wasn't being used -- we have a home.js that does the home page stuff